lib/zones.js

```diff
@@ -45,6 +45,21 @@ function read(zid, options) {
   });
 }
 
+function edit(zid, options) {
+  if (zid instanceof Zone) {
+    zid = zid.id;
+  }
+
+  var uri = join('zones', zid);
+
+  options.method = 'PATCH';
+  options.body = JSON.stringify(options.body);
```

Comment:
This means using the API will look something like this:
```
client.editZone(zone, {
    body: {
      paused:false
    }
  });
```
I'm not sure if I like that or not. Thoughts?